addons/website_forum/views/forum_forum_templates_forum_all.xml

```diff
@@ -106,18 +88,13 @@
 <template id="forum_all_oe_structure_forum_all_top" inherit_id="website_forum.forum_all" name="Forum Navigation (oe_structure_forum_all_top)">
     <xpath expr="//*[hasclass('oe_structure')][@id='oe_structure_forum_all_top']" position="replace">
         <div class="oe_structure oe_empty" id="oe_structure_forum_all_top">
-            <section class="s_cover parallax s_parallax_is_fixed bg-black-50 py-5" data-scroll-background-ratio="1" data-snippet="s_cover">
-                <span class="s_parallax_bg oe_img_bg" style="background-image: url('/web/image/website.s_cover_default_image'); background-position: 50% 0;"/>
-                <div class="o_we_bg_filter bg-black-50"/>
-                <div class="container">
-                    <div class="row s_nb_column_fixed">
-                        <div class="col-lg-12">
-                            <h1 class="o_default_snippet_text text-center">Our forums</h1>
-                            <p class="lead o_default_snippet_text mb-0" style="text-align: center;">
-                                This community is for professional and enthusiast users, partners and programmers.
-                            </p>
-                        </div>
-                    </div>
+            <section class="s_cover o_colored_level s_parallax_no_overflow_hidden pt96 pb48" data-scroll-background-ratio="0" data-snippet="s_cover" data-name="Cover" style="">
+                <div class="s_allow_columns container">
+                    <h1 style="text-align: center;"><font style="font-size: 62px; font-weight: bold;">Community Forums</font></h1>
+                    <p class="lead" style="text-align: center;">Tap into the collective knowledge of our community by asking your questions in our forums,<br/> where helpful members are ready to assist you.</p>
```

Comment:
Hi @std-odoo , in this particular case it's one of web-editor's snippets, if we use bootstrap classes it can't be edited by the user in the web-editor :)

Reply:
ah ok :) thanks for explanation